solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -2211,6 +2225,15 @@ public void checkOverseerDesignate() {
     }
   }
 
+  public void setPreferredOverseer() throws KeeperException, InterruptedException {
+    ZkNodeProps props = new ZkNodeProps(Overseer.QUEUE_OPERATION, ADDROLE.toString().toLowerCase(Locale.ROOT),
+        "node", getNodeName(),
+        "role", "overseer",
+        "persist", "false");
+    log.info("Going to add role {} ", props);
```

Comment:
Should we warn that this is deprecated and point people to the new way to set roles?

Reply:
Fixed, thanks.